recipes/opentrackio-cpp/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+class PackageConan(ConanFile):
+    name = "opentrackio-cpp"
+    description = "A Cpp helper library for usage with the OpenTrackIO protocol."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mosys/opentrackio-cpp"
+    topics = ("opentrackio", "smpte")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "15",
+            "clang": "14",
+            "gcc": "13",
+            "msvc": "193",
+            "Visual Studio": "20",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            # Nothing is exported from the library, so we can't use it as a shared library
+            del self.options.shared
+            self.package_type = "static-library"
```

Comment:
@mosys I hardcoded Windows builds to only be static, as nothing gets exported in a shared build and thus the library is not present in those cases. Do you have any insight if this is an expected case? Thanks!